terra2/assetlist.json

```diff
@@ -544,6 +544,31 @@
         }
       ]
     },
+    {
+      "description": "ERIS Liquid Alliance Staked ampROAR",
```

Comment:
Please move this new asset object to the bottom of the assets array.

Reply:
done